### PR TITLE
Partial Translation of README.md to Filipino

### PR DESCRIPTION
**Pull request by caratzky**
Good day, @DanielRosenwasser !

This file is composed of about 2600 translatable words, so I decided to split them into two. I'll be submitting another pull request updating this file, this time the whole file being completely translated. I hope you merge this one as soon as possible so I can edit it in my next pull request. 

I'm also doing this because in the rewarding rules of Utopian, only a minimum of 1000 words is needed. So I'm planning to split these into two to receive two separate rewards. For this commit, I've translated about 1400 words. Hope you get my point. Thank you. 

Best Regards,
@caratzky

### CODE REVIEW

File: README.fil.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+Kung hindi mo pa rin ito mahanap, tingnan kung ito ay [mabibigkis](http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) sa sarili nitong mga pagta-type.
+Karaniwan itong ibinibigay sa isang `"types"` r `"typings"` na field sa `package.json`,
+o maghanap lang ng kahit anong ".d.ts" na mga file sa pakete at manu-manong isali sila kasama ang isang `/// <reference path="" />`.
+
+
+### Ibang mga pamamaraan
+
+Pwede silang magagamit ng TypeScript 1.0.
+
+* [Mga Pagta-tyoe](https://github.com/typings/typings)
```
Comment (DanielRosenwasser) [inline README.fil.md]: This should be left as `typings`

File: README.fil.md [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+#### Baguhin ang isang umiiral na pakete
+
+* `cd types/my-package-to-edit`
+* Gumawa ng mga pagbabago. Tandaang baguhin ang mga pagsusuri.
+* Baka gusto mo ring idagdag ang iyong sarili sa seksyong "Mga Paglalarawan ni" ng header ng pakete.
+  - Ito ang magpapaalam sa iyo (sa pamamagitan ng iyong GitHub na username) kapag may gagawa ng isang pull request or isyu tungkol sa pakete.
+  - Gawin ito sa pamamagitan ng pagdagdag ng iyong pangalan sa katapusan ng linya, katulad ng `// Mga Paglalarawan ni: Alice <https://github.com/alice>, Bob <https://github.com/bob>`.
+  - O kapag may mas marami pang tao, pwede itong maging maramihang linya.
+  ```typescript
+  // Mga Paglalarawan ni: Alice <https://github.com/alice>
```
Comment (DanielRosenwasser) [inline README.fil.md]: This needs to be left with the old text for contributors

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
Hey @caratzky, thanks so much for the PR! I was unable to find reviewers, however, I ran this through a translator and this looks pretty darn accurate!

If you wouldn't mind making updates based on my feedback and pinging me, I can then pull this in. Thanks!!

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has unaddressed comments and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**